src/rastervision/runner/experiment_runner.py

```diff
@@ -1,37 +1,38 @@
 from abc import (ABC, abstractmethod)
 from typing import (List, Union)
+import logging
 
-import click
 from google.protobuf import json_format
 
 import rastervision as rv
 from rastervision.runner import (CommandDefinition, CommandDAG)
 from rastervision.cli import Verbosity
 
+log = logging.getLogger(__name__)
```

Comment:
Experiment Runner output is similar to the CLI output - the output is meant for the console, and the coloring is important. This should be moved back to the echo commands.